packages/playground/cli/src/test/test-running-unbuilt-cli.sh

```diff
@@ -53,6 +54,7 @@ function test_playground_cli_multi_worker() {
 
 	# TODO: Also test with asyncify once we multiple workers there.
```

Comment:
My typo from an earlier PR.
```suggestion
	# TODO: Also test with asyncify once we support multiple workers there.
```